### PR TITLE
SISRP-5790 - Front End Dev: Alerts and Notifications Tab - bConnected Card

### PR DESCRIPTION
**Pull request by winstonkim**
https://jira.berkeley.edu/browse/SISRP-5790

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/pages/profileController.js [hunk 1]
```diff
@@ -19,15 +19,34 @@ angular.module('calcentral.controllers').controller('ProfileController', functio
...
+  };
+
+  /**
    * Find the category object when we get a categoryId back
    */
   var findCategory = function(categoryId) {
-    return _.find(navigation[0].categories, function(category) {
-      return category.id === categoryId;
-    });
+    for (var i = 0; i < navigation.length; i++) {
```
Comment (christianv) [inline src/assets/javascripts/angular/controllers/pages/profileController.js]: @winstonkim took me a while but could you test it with the following code instead:

```javascript
_.find(_.flatten(_.pluck(navigation, 'categories')), {
  id: categoryId
})
```

Comment (winstonkim) [inline src/assets/javascripts/angular/controllers/pages/profileController.js]: @christianv that works! i'll make another commit with that change. thanks!

File: src/assets/templates/widgets/profile/bconnected.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div data-ng-controller="bConnectedController">
+  <section data-ng-if='connectedServices.length' class="cc-settings-section row collapse">
+      <div data-ng-repeat="service in connectedServices">
```
Comment (christianv) [inline src/assets/templates/widgets/profile/bconnected.html]: @winstonkim indentation looks a bit off here in this file, could you have a look?

**Comment by christianv** [comment]
Apart from the indentation fix, this lgtm

File: src/assets/templates/widgets/profile/bconnected.html (hunk 2 above)
Comment (winstonkim) [inline src/assets/templates/widgets/profile/bconnected.html]: @christianv ok fixed!

**Pull request merged**